README.md

```diff
@@ -1,73 +1,137 @@
- * authentication and access control
- * the database is just a C extension to the
-   application server and can be turned off
+## Getting started
 
-Supported platforms are Linux/x86 and FreeBSD/x86, Mac OS X.
+### Get hold of Tarantool
 
-Tarantool is ideal for data-enriched components of
-scalable Web architecture: queue servers, caches,
-stateful Web applications.
+* Try online:
 
-To download and install Tarantool as a binary package for your OS, please visit
-https://tarantool.io/en/download/.
+  Visit our playground at https://www.tarantool.io/en/try-dev/
 
-To build Tarantool from source, see detailed instructions in the Tarantool
-documentation at https://tarantool.io/en/doc/2.1/dev_guide/building_from_source/.
+* Use a Docker image:
 
-Please report bugs at https://github.com/tarantool/tarantool/issues
-We also warmly welcome your feedback in the discussion mailing
-list, tarantool@googlegroups.com.
+  ```bash
+  docker run -d -p 3301:3301 -v $(pwd):/var/lib/tarantool tarantool/tarantool:2
+  ```
 
-Thank you for your interest in Tarantool!
+* Install locally:
+
+  * On Ubuntu/Fedora/Debian/Centos:
+
+    ```bash
+    curl -L http://tarantool.io/installer.sh | VER=2.2 sudo bash
+    ```
+
+  * On Mac OS using [Homebrew](https://brew.sh):
+
+    ```bash
+    brew install tarantool
+    ```
+
+  For more details, please visit https://tarantool.io/download.
+
+* Build from source:
+
+  Please see
+  [detailed instructions](https://tarantool.io/doc/2.2/dev_guide/building_from_source/)
+  in the Tarantool documentation.
+
+### Connectors
+
+We have connectors for many popular languages:
+
+* [Python](https://github.com/igorcoding/asynctnt)
+* [PHP](https://github.com/tarantool-php/client)
+* [Java](http://github.com/tarantool/tarantool-java/)
+* [Go](https://github.com/tarantool/go-tarantool)
+
+[Check out connector details for other languages](https://www.tarantool.io/doc/2.2/book/connectors/)
+and connect to your current instance of Tarantool.
+
+## Use cases
+
+* Transactional cache
+* Persistent queues
+* ETL
+* Base for IoT platform
+
+## Ready-made examples
+
+We have a frawework for creating of distributed apps called [Tarantool Cartridge](https://github.com/tarantool/cartridge).
+See the [tarantool/examples](https://github.com/tarantool/examples) repository
+with source code of typical projects based on this framework.
+
+## Who uses Tarantool
+
+* Nokia uses Tarantool as part of their IoT platform
+  (in-memory database + Lua scripting)
+* Russian Enterprise: banks, telecoms, retails
+* Mail.Ru, a major Russian Internet company, uses Tarantool for its email
+  service, ad systems, social services, media projects
+* Mastercard
+* Avito, Russian largest classified advertisements website
+* and many other companies
+
+## Community
+
+Join our Telegram [news channel](https://t.me/tarantool_news) and chats:
+
+* in [Russian](https://teleg.run/tarantoolru)
+* in [English](https://teleg.run/tarantool)
```

Comment:
Same question as below: `https://teleg.run` seems to be the third-party website. Why should we trust it so much to give links to it from our front page? What is wrong with just `https://t.me/tarantool` and `https://t.me/tarantoolru`?